src/pudl/transform/eia861.py

```diff
@@ -558,17 +558,20 @@ def _tidy_class_dfs(df, df_name, idx_cols, class_list, class_type):
         data_cols.stack(level=0, dropna=False)
         .reset_index()
     )
-    denorm_cols = _filter_non_class_cols(
-        raw_df, class_list).reset_index()
+    denorm_cols = _filter_non_class_cols(raw_df, class_list).reset_index()
 
     # Merge the index, data, and denormalized columns back together
     tidy_df = pd.merge(denorm_cols, data_cols, on=idx_cols)
 
+    # Compare reported totals with sum of component columns
+    _compare_totals(data_cols, idx_cols, class_type, df_name)
+
     # Remove the now redundant "Total" records -- they can be reconstructed
     # from the other customer classes.
-    # tidy_df = tidy_df.query(f"{class_type}!='total'")
+    if keep_totals is False:
```

Comment:
this is a vv small thing but I think it is best practice to say `if not boolean_thing` instead of `if boolean_thing is False`

Reply:
Good to know!! thanks